<!--
@Author: Jose Rojas
@Date:   2018-07-10T08:01:12-07:00
@Email:  jrojas@redlinesolutions.co
@Project: ros-libav-node
@Last modified by:   jrojas
@Last modified time: 2018-07-10T09:42:59-07:00
@License: MIT License
@Copyright: Copyright @ 2018, Jose Rojas
-->

<launch>
    <node pkg="ros_libav" type="record.py" name="libav_video_recorder" >
      <param name="libav_video_encoder_image_filetype" value="mpg" />
      <param name="libav_video_encoder_image_topic" value="rgb" />
      <param name="libav_video_encoder_stream_topic" value="rgb/stream" />
      <param name="libav_video_encoder_stream_event_topic" value="rgb/stream/event" />
      <param name="libav_video_encoder_delay" value="0.5" />
      <param name="libav_video_encoder_finalize_delay" value="0.75" />
    </node>
</launch>
